.changeset/nine-actors-fly.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/core-components': patch
+---
+
+Add an aria-label to the support button to improve accesibility for screen readers
```

Comment:
Looks like there's a spelling mistake here for `accesibility`

Reply:
Thanks for that, should be fixed now!